.ci/magician/cmd/generate_comment.go

```diff
@@ -71,6 +69,14 @@ var generateCommentCmd = &cobra.Command{
 			env[ev] = val
 		}
 
+		for _, tokenName := range []string{"GITHUB_TOKEN_DOWNSTREAMS", "GITHUB_TOKEN_MAGIC_MODULES"} {
+			val, ok := lookupGithubTokenOrFallback(tokenName)
+			if !ok {
+				fmt.Printf("Did not provide %s or GITHUB_TOKEN environment variable\n", tokenName)
+				os.Exit(1)
+			}
+			env[tokenName] = val
```

Comment:
As written this will fail if GITHUB_TOKEN_DOWNSTREAMS and GITHUB_TOKEN are both unset (even if 
GITHUB_TOKEN_MAGIC_MODULES is set)

```suggestion
			if ok {
				env[tokenName] = val
				break
			}
```